### PR TITLE
Fix Lava Testnet pre-fork Instance

### PR DESCRIPTION
**Pull request by JeremyParish69**
Fix Lava Testnet pre-fork Instance
The shouldn't be a lavatestnet1 (old) and lavatestnet2 (new), but instead:
lavatestnet (current, and stays current), and lavatestnet1 (the first archived pre-fork instance)

### CODE REVIEW

**Comment by JeremyParish69** [comment]
@omerlavanet can I please get a thumbs up on this PR?

File: testnets/lavatestnet/chain.json [hunk 1]
```diff
@@ -36,49 +37,109 @@
...
+    "cosmos_sdk_version": "0.47.6",
     "consensus": {
       "type": "tendermint",
-      "version": "0.34"
+      "version": "0.37"
     },
     "cosmwasm_version": "0.30",
     "cosmwasm_enabled": false,
-    "ibc_go_version": "v7.2.0",
+    "ibc_go_version": "3.0.1",
```
Comment (Yaroms) [inline testnets/lavatestnet/chain.json:54]: we use ibc v7

File: testnets/lavatestnet/chain.json [hunk 2]
```diff
@@ -36,49 +37,109 @@
...
+        "name": "v0.31.1",
+        "tag": "v0.31.1",
+        "height": 675100,
+        "next_version_name": ""
+      },
+      {
+        "name": "v0.32.0",
+        "tag": "v0.32.0",
+        "height": 711251,
+        "recommended_version": "v0.32.0",
```
Comment (Yaroms) [inline testnets/lavatestnet/chain.json:127]: our latest version is v1.0.1

**Comment by Yaroms** [comment]
Hey @JeremyParish69 we will soon fix the PR and create a new one, thank you for the help

**Pull request closed without merging**